tt_eager/tt_dnn/op_library/tilize/kernels/dataflow/reader_unary_pad_dims_split_rows_multicore.cpp

```diff
@@ -0,0 +1,96 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+
+#include "dataflow_api.h"
+
+#define ALWI inline __attribute__((always_inline))
```

Comment:
dataflow_api.h defines this as FORCE_INLINE already (not entirely sure why data movement vs compute uses different names for this though)